packages/core/src/lib/command-utils.ts

```diff
@@ -45,18 +45,16 @@ export function resolveNetworkUrl(config: SwankyConfig, networkName: string): st
 
 export async function storeArtifacts(
   artifactsPath: string,
-  contractName: string,
+  contractName: string
 ): Promise<BuildData> {
   const ts = Date.now();
   const fullPath = path.resolve(STORED_ARTIFACTS_PATH, contractName, ts.toString());
   const relativePath = path.relative(path.resolve(), fullPath);
   const buildData = {
-    timestamp: ts,
     artifactsPath: relativePath,
+    timestamp: Date.now(),
   };
 
-  await fs.ensureDir(buildData.artifactsPath);
```

Comment:
Why removed?
Removal of this line started to cause errors.
<img width="1512" alt="Screen Shot 2023-05-10 at 18 16 06" src="https://github.com/AstarNetwork/swanky-cli/assets/28953860/3646c306-6cdb-4982-830a-5fc00339addb">

Reply:
I removed the buildData history completely. (this stayed by mistake, or merge oversight)
Now all the contracts are expected to have artifacts in `artifacts/contract` directory.